src/types.ts

```diff
@@ -147,4 +147,52 @@ export interface ManifestOptions {
    * @default A combination of `routerBase` and `options.build.publicPath`
    */
   publicPath: string
+  /**
+   * @default []
+   */
+  related_applications: {
+    platform: string,
+    url: string,
+    id?: string,
+  }[]
+  /**
+   * @default false
+   */
+  prefer_related_applications: boolean
```

Comment:
should be `prefer_related_applications?: boolean` ([spec](https://developer.mozilla.org/en-US/docs/Web/Manifest/prefer_related_applications))

Reply:
yes true sorry!